docs/src/getting-started-vscode-js.md

```diff
@@ -64,27 +64,29 @@ With the VS Code extension you can debug your tests right in VS Code see error m
 
-### Run in Debug Mode
+### Live Debugging
 
-To set a breakpoint click next to the line number where you want the breakpoint to be until a red dot appears. Run the tests in debug mode by right clicking on the line next to the test you want to run. A browser window will open and the test will run and pause at where the breakpoint is set.
+You can debug your test live in VS Code. After running a test with the 'show browser' option checked, click on any of the locators in VS Code and it will be highlighted in the Browser window. You can also edit the locators in VS Code and Playwright will highlight it if it exits and show you if there is more than one result.
 
-<img width="1149" alt="setting debug test mode" src="https://user-images.githubusercontent.com/13063165/197715919-98f32957-2ae1-478b-9588-d93cc4548c67.png" />
+<img width="1394" alt="Screenshot 2023-01-16 at 18 43 10" src="https://user-images.githubusercontent.com/13063165/212738833-5f9bcbb2-0f58-4664-b6ea-fccbb6f26666.png" />
 
+### Run in Debug Mode
 
-### Live Debugging
+To set a breakpoint click next to the line number where you want the breakpoint to be until a red dot appears. Run the tests in debug mode by right clicking on the line next to the test you want to run. 
 
-You can modify your test right in VS Code while debugging and Playwright will highlight the selector in the browser. This is a great way of seeing if the selector exits or if there is more than one result. You can step through the tests, pause the test and rerun the tests from the menu in VS Code.
+<img width="1269" alt="setting debug test mode" src="https://user-images.githubusercontent.com/13063165/212739847-ecb7dcfe-8929-45f3-b24e-f9c4b592f430.png" />
 
+A browser window will open and the test will run and pause at where the breakpoint is set. You can step through the tests, pause the test and rerun the tests from the menu in VS Code.
 
-<img width="1350" alt="Live Debugging in VS Code" src="https://user-images.githubusercontent.com/13063165/197967885-512df81f-12e3-45e5-b90f-42ed0f064eac.png" />
+<img width="1269" alt="running test in debug mode" src="https://user-images.githubusercontent.com/13063165/212740233-3f278825-13e7-4a88-a118-dd4478d43a16.png" />
 
 ### Debug in different Browsers
 
 Debug your tests on specific browsers by selecting a profile from the dropdown. Set the default profile or select more than one profile to debug various profiles. Playwright will launch the first profile and once finished debugging it will then launch the next one.
 
-<img width="1221" alt="debugging on specific profile" src="https://user-images.githubusercontent.com/13063165/197738552-06aa8a83-6a6b-4aad-ab23-d449640e1f5f.png" />
+<img width="1269" alt="debugging on specific profile" src="https://user-images.githubusercontent.com/13063165/212740442-5c19331a-5de7-4bc5-a98d-d5f465d42c42.png" />
```

Comment:
I think usually they don't want to debug all the tests, they want to debug single tests in different browers. This means:
1. Change the default debugging profile to e.g. firefox
2. Click on a test, right click it and say debug.